api/query.go

```diff
@@ -244,7 +251,7 @@ func (e *Endpoint) FenceGet(ctx *gin.Context) {
 // @Param    body  body  FenceRequest  True  "Request Parameters"
 // @Accept   application/json
 // @Produce  multipart/mixed
-// @Success  200
+// @Success  200 {object} vds.FenceMetadata
```

Comment:
Maybe add a comment that FenceMetadata only applies to the first part in the response? See slice equivalent